### PR TITLE
Add InstallOperator fucntion

### PR DESCRIPTION
**Pull request by MarcelMue**
Towards https://github.com/giantswarm/giantswarm/issues/4355.
Towards https://github.com/giantswarm/giantswarm/issues/4337.

### CODE REVIEW

File: pkg/release/release.go [hunk 1]
```diff
@@ -20,18 +23,22 @@ const (
 
 type Config struct {
 	ApprClient *apprclient.Client
+	FileLogger *filelogger.FileLogger
 	HelmClient *helmclient.Client
+	K8sClient  kubernetes.Interface
 	Logger     micrologger.Logger
 
-	Namespace string
+	TargetNamespace string
```
Comment (kopiczko) [inline pkg/release/release.go]: What's the reason behing calling it a target namespace? For me `Namespace` makes total sense.

Comment (MarcelMue) [reply pkg/release/release.go]: Because this is the target of our installation of resources, I can revert back to namespace if you prefer.

Comment (kopiczko) [reply pkg/release/release.go]: This is kind of obvious that this is the namespace where releases will be installed, isn't it?
I'd prefer to call it a `Namespace` yes, but no blocker.

Comment (MarcelMue) [reply pkg/release/release.go]: Reverted back to `namespace` PTAL.

**Review by kopiczko — APPROVED** [review]
LGTM.

**Review by kopiczko — CHANGES_REQUESTED** [review]
https://github.com/giantswarm/e2e-harness/pull/145/files#r224035185

File: pkg/release/condition_set.go [hunk 2]
```diff
@@ -48,7 +48,6 @@ func (c *conditionSet) CRD(ctx context.Context, name string) conditionFn {
 			} else {
 				return backoff.Permanent(microerror.Mask(err))
 			}
-			return nil
```
[the remark this answers is not in the document]
Comment (MarcelMue) [reply pkg/release/condition_set.go]: Added back, the line was unreachable but the problem was the if statement in reality. Fixed it now.

File: pkg/release/condition_set.go [hunk 3]
```diff
@@ -45,7 +45,7 @@ func (c *conditionSet) CRD(ctx context.Context, name string) conditionFn {
 			_, err := c.extClient.ApiextensionsV1beta1().CustomResourceDefinitions().Get(name, metav1.GetOptions{})
 			if apierrors.IsNotFound(err) {
 				return microerror.Mask(err)
-			} else {
+			} else if err != nil {
```
Comment (kopiczko) [inline pkg/release/condition_set.go]: Good catch.

**Review by kopiczko — APPROVED** [review]

**Comment by MarcelMue** [comment]
Going with this, let's iterate.

**Pull request merged**